main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -74,6 +74,8 @@ HistoryPanel.prototype._render = function() {
   this._div.empty().unbind().html(DOM.loadHTML("core", "scripts/project/history-panel.html"));
 
   var elmts = DOM.bind(this._div);
+
+  HistoryPanel.prototype.showHideHistoryFacetSplash();
```

Comment:
```suggestion
  HistoryPanel.showHideSplash();
```
2 things here. 
1. History is not needed as it's in the parent name, and there is no link at all with facets.
2. The `HistoryPanel.prototype` syntax is necessary when creating/assigning the value, but not when calling the static function (as it is inherited). Just shorter and clearer.

Reply:
@antoine2711: I just did a quick user test and `HistoryPanel.showHideSplash()` is not recognized as a function but `HistoryPanel.prototype.showHideHistoryFacetSplash()` is